### ISSUE TITLE
Remove hardcoded paths in salt minion/master confs

### ISSUE THREAD

**Issue by sayf-eddine-scality** [scality/metalk8s#651]
the path that points to the fileroots, pillar and states should not be hardcoded and should depend on the metalk8s version

**Comment by jbertran** [comment]
Fileroots tackled in #646

**Comment by jbertran** [comment]
Bumping this to bug, as this makes the bootstrap fail on some stuff now that the expected mountpoint is version-dependent.

Offending statements:

```
salt/metalk8s/registry/populated.sls:35:{% set images_path = '/srv/scality/metalk8s-dev/images' %}
salt/metalk8s/registry/populated.sls:45:    - archive_path: /srv/scality/metalk8s-dev/images/{{ image.name }}-{{ image.tag }}.tar.gz
salt/metalk8s/registry/init.sls:9:    - archive_path: /srv/scality/metalk8s-dev/images/registry-2.7.1.tar
salt/metalk8s/containerd/configured.sls:3:    '/srv/scality/metalk8s-dev/salt/metalk8s/containerd/files/pause-3.1.tar'
```

**Comment by jbertran** [comment]
Current state: in salt files, the version is hard-coded to the correct version (`2.0`). This should depend on short_version instead (as generated in `product.txt`), probably as a custom module whose return values are rendered on build.

We should wait for the build system refactor for this one.

**Comment by gdemonet** [comment]
The `{{ saltenv }}` variable available in SLS files will return what is needed, so we should be able to avoid the need for a custom module. PoC incoming.

**Comment by gdemonet** [comment]
Closed by #761
